ci/test-flow-visibility.sh

```diff
@@ -0,0 +1,145 @@
+
+Setup Grafana, ClickHouse and Antrea Flow Aggregator to validate flow-aggregator.yml and flow-visibility.yml.
+
+    --kubeconfig    Path to kubeconfig.
+
+"
+
+echoerr() {
+  >&2 echo "$@"
+}
+
+print_usage() {
+  echoerr "$_usage"
+}
+
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+case $key in
+    --kubeconfig)
+    export KUBECONFIG="$2"
+    shift 2
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+start_antrea() {
+  echo "=== Starting Antrea ==="
+  kubectl apply -f ${GIT_CHECKOUT_DIR}/build/yamls/antrea.yml
+  kubectl rollout status --timeout=5m deployment/coredns -n kube-system
+  kubectl rollout status --timeout=5m deployment.apps/antrea-controller -n kube-system
+  kubectl rollout status --timeout=5m daemonset/antrea-agent -n kube-system
+}
+
+setup_flow_visibility() {
+  echo "=== Starting Flow Visibility ==="
+  # install ClickHouse operator
+  kubectl apply -f ${GIT_CHECKOUT_DIR}/build/yamls/clickhouse-operator-install-bundle.yml
+  kubectl apply -f ${GIT_CHECKOUT_DIR}/build/yamls/flow-visibility.yml
+  echo "=== Waiting for ClickHouse and Grafana to be ready ==="
+  sleep 15
+  kubectl wait --for=condition=ready pod -l app=clickhouse-operator -n kube-system --timeout=600s
+  kubectl wait --for=condition=ready pod -l app=clickhouse -n flow-visibility --timeout=600s
+  kubectl wait --for=condition=ready pod -l app=grafana -n flow-visibility --timeout=600s
+}
+
+check_grafana() {
+  echo "=== Check the installation of Grafana plugins ==="
+  grafana_logs=$(kubectl logs -n flow-visibility $(kubectl -n flow-visibility get pod -l app=grafana -o jsonpath="{.items[0].metadata.name}"))
+  if ( echo ${grafana_logs} | grep -q 'msg="Plugin registered" logger=plugin.manager pluginId=antreaflowvisibility-grafana-sankey-plugin' ); then
+    echo "=== antreaflowvisibility-grafana-sankey-plugin installed correctly ==="
+  else
+    echo "=== antreaflowvisibility-grafana-sankey-plugin is NOT installed correctly ==="
+    FAILURE=true
+  fi
+  if ( echo ${grafana_logs} | grep -q 'msg="Plugin registered" logger=plugin.manager pluginId=grafana-clickhouse-datasource' ); then
+    echo "=== grafana-clickhouse-datasource installed correctly ==="
+  else
+    echo "=== grafana-clickhouse-datasource is NOT installed correctly ==="
+    FAILURE=true
+  fi
+}
+
+config_antrea() {
+  echo "=== Stopping Antrea === "
+  kubectl delete -f ${GIT_CHECKOUT_DIR}/build/yamls/antrea.yml
+  echo "=== Enable Antrea Flow Exporter ==="
+  sed -i -e "s/#  FlowExporter: false/  FlowExporter: true/g" ${GIT_CHECKOUT_DIR}/build/yamls/antrea.yml
+}
+
+setup_flow_aggregator() {
+  echo "=== Config Antrea Flow Aggregator ==="
+  perl -i -p0e 's/      # Enable is the switch to enable exporting flow records to ClickHouse.\n      #enable: false/      # Enable is the switch to enable exporting flow records to ClickHouse.\n      enable: true/' ./build/yamls/flow-aggregator.yml
```

Comment:
not sure why we need to use perl here instead of sed
also is this not possible by using `hack/generate-manifest-flow-aggregator.sh`?

Reply:
Since there are two lines of `#enable: false` in `flow-aggregator.yml` right now for FlowCollector and Clickhouse respectively, using `perl` seems better and easier to do this multi-line string replacement than sed. 
`hack/generate-manifest-flow-aggregator.sh` is a good point, looks like it has not been updated with our new config in Flow Aggregator https://github.com/antrea-io/antrea/blob/main/hack/generate-manifest-flow-aggregator.sh#L141. Do you want me to update that .sh in this PR or open another one after?

Reply:
You can look into this in a follow up PR, let me approve this one